webui/src/components/BugTitleForm/SetTitle.graphql

```diff
@@ -0,0 +1,7 @@
+mutation setTitle($input: SetTitleInput!) {
+  setTitle(input: $input) {
+      bug {
+        humanId
```

Comment:
you should retrieve the complete `id` here. `humanId` is a truncated Id, that's not enough to update the cache.

Reply:
Oh! I´ll fix it!